plugins/woocommerce/client/admin/client/embedded-body-layout/embedded-body-layout.tsx

```diff
@@ -28,7 +27,6 @@ function isWPPage(
 }
 
 const EMBEDDED_BODY_COMPONENT_LIST: React.ElementType[] = [
-	PaymentRecommendations,
```

Comment:
hide payment recommendations embedded on the bottom of the settings page. is it the right way?

Reply:
I think so - but we may need to check if there are any other pages in which we show this `PaymentRecommendations` embed where we still want to show it? Maybe Ghidorah will know more about how this works, it would be good to get their opinion here 🙂

Reply:
I got tests failing, not sure if I did it right, I decided not including this change, better doing it properly in a separate small PR